src/window_battlestatus.cpp

```diff
@@ -185,7 +186,12 @@ void Window_BattleStatus::DrawGaugeSystem2(int x, int y, int cur_value, int max_
 		gauge_width = 25 * cur_value / max_value;
 	}
 
-	contents->StretchBlit(Rect(x, y, gauge_width, 16), *system2, Rect(48 + gauge_x, 32 + 16 * which, 16, 16), Opacity::Opaque());
+	if (gauge_width >= 13) {
+		contents->StretchBlit(Rect(x, y, 13, 16), *system2, Rect(48 + gauge_x, 32 + 16 * which, 8, 16), Opacity::Opaque());
+		contents->StretchBlit(Rect(x + 13, y, gauge_width - 13, 16), *system2, Rect(56 + gauge_x, 32 + 16 * which, 8, 16), Opacity::Opaque());
+	} else {
+		contents->StretchBlit(Rect(x, y, gauge_width, 16), *system2, Rect(48 + gauge_x, 32 + 16 * which, 8, 16), Opacity::Opaque());
```

Comment:
> Battle 2k3: Change gauge display 

Can you explain what the magic number 13 means here?
Also for what is the pixman expert?

Reply:
I admit this one is more of a hack. I take the left 8 pixels of the gauge and stretch them to 13 pixels, and the right 8 pixels of the gauge gets stretched to 12 pixels, which sums to 25 pixels like in RPG_RT. By putting the two halves together you get a more accurate output than directly stretch the entire gauge from 16 to 25 pixels. The so-called "pixman expert" is somebody we need to refactor the StretchBlit and similar functions to match RPG_RT behavior (see issue #446). As far as I know we are using pixman for the pictures, so I went with the term "pixman expert" for people who are experienced in using this library. (This entire stuff here is kinda hard to explain...)

Reply:
Can I suggest something?
Could you try to replicate a RPG2k3 gauge using operations (stretch) in a normal image editor like Gimp or Paint.net?
When the exact operations are known this is easier to replicate in the Player instead of experimenting.

Reply:
But with gimp you get the result you want?

Reply:
No.

Reply:
Alternatively we can remove the gauge stuff from this PR and wait until #446 is fixed (this should automatically fix the gauge display).

Reply:
Okay. Please remove this commit then. As this is not a 100% fix I do not think that making the code harder to read is worth it here. Sorry

Reply:
Removed the gauge commit now.